argo_jupyter_scheduler/utils.py

```diff
@@ -165,16 +170,25 @@ def gen_papermill_command_input(
     output_path = gen_output_path(input_path)
     log_path = gen_log_path(input_path)
     conda_env_path = gen_conda_env_path(conda_env_name, use_conda_store_env)
+    html_path = gen_html_path(input_path)
 
     logger.info(f"conda_env_path: {conda_env_path}")
     logger.info(f"output_path: {output_path}")
     logger.info(f"log_path: {log_path}")
-
-    return [
-        f"conda run -p {conda_env_path} papermill -k {kernel_name} {input_path} {output_path}",
-        "&>",
-        log_path,
-    ]
+    logger.info(f"html_path: {html_path}")
+
+    return (
+        f"conda run -p {conda_env_path} /bin/sh -c "
+        "\"{ "
+        f"papermill -k {kernel_name} {input_path} {output_path}"
+        " && "
+        f"jupyter nbconvert --to html {output_path} --output {html_path}"
+        " ; "
+        " } "
+        " > "
+        f"{log_path}"
+        " 2>&1\""
```

Comment:
This also fixes a bug with redirection. Previously, code relied on `&>`, which is a Bash feature, and that didn't work because this is executed via /bin/sh.

Reply:
If we can split this into variables for multiple commands (like `conda run .. pappermill_cmd && jupyter_nbconvert_cmd`), then it would more readable.